bin/node/runtime/Cargo.toml

```diff
@@ -41,6 +41,7 @@ sp-transaction-pool = { version = "4.0.0-dev", default-features = false, path =
 sp-version = { version = "4.0.0-dev", default-features = false, path = "../../../primitives/version" }
 sp-npos-elections = { version = "4.0.0-dev", default-features = false, path = "../../../primitives/npos-elections" }
 sp-io = { version = "4.0.0-dev", default-features = false, path = "../../../primitives/io" }
+sp-sandbox = { version = "0.10.0-dev", default-features = false, path = "../../../primitives/sandbox" }
```

Comment:
Why do we need this?

Reply:
It was introduced as a part of a [fix](https://github.com/paritytech/substrate/pull/10268/files).
The second part `wasmer-sandbox = ["sp-sandbox/wasmer-sandbox"]` was removed because of a note "Don't use that on a production chain.".

Looks like we can remove `sp-sandbox`

Reply:
Let's remove.

Reply:
Removed